### ISSUE TITLE
DPL-090 As a Sample Management user, I would like to be able to carry out 're-racking' of tube racks from within Sequencescape, rather than in Samples Extraction (C=?, V=?)

### ISSUE THREAD

**Issue by KatyTaylor** [sanger/sequencescape#3260]
**User story**
As a Sample Management user, I would like to be able to carry out 're-racking' of tube racks from within Sequencescape, rather than in Samples Extraction. The current method in Samples Extraction is not user friendly (see https://github.com/sanger/samples_extraction/issues/137 and SOP attached to it) and takes 45 mins-1 hr per rack. By having it in Sequencescape, we can also take advantage of the new(ish) ability to represent tube racks as tube racks, rather than as plates.

**Who are the primary contacts for this story**
Danni, Katy, Eduardo, Andrew

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] I can create a new tube rack in Sequencescape, made up of tubes from existing tube racks in Sequencescape.
- [ ] I can provide the information about which tubes are in which places in the rack by uploading a csv from the rack scanner.
- [ ] I can choose to create a plate rather than a tube rack, to maintain backwards compatibility with the current way of doing things, while we improve the rest of the tube rack functionality.

References
This story could be done as an alternative, or a follow on improvement from https://github.com/sanger/samples_extraction/issues/137. (It can also be run in parallel to the Samples Extraction functionality if needed)

To be useful in current Sample Management (e.g. DDD) and Cardinal contexts, another story about stamping from plate to tube rack will also be needed - https://github.com/sanger/sequencescape/issues/3261

**Additional context**
UX suggestions here - https://lucid.app/lucidchart/invitations/accept/inv_adf174f9-fbb4-4f4a-b58f-21dc7da6564b?viewport_loc=-45%2C449%2C2219%2C1237%2C0_0

This feature has come up recently because it will also be used in the Cardinal pipeline. It is thought that improved tube rack / re-racking functionality could also be useful for the Tree of Life project, and Long Read, as they both use tubes currently.

Previous tube rack related questions and decisions - https://ssg-confluence.internal.sanger.ac.uk/pages/viewpage.action?pageId=101361052

**Comment by SujitDey2022** [comment]
@KatyTaylor could you help review this story and confirm if this is still relevant or can be archived?

**Comment by stevieing** [comment]
This was discussed with sample prep and is no longer used so can be deprecated.
